components/Navbar.tsx

```
@@ -1,15 +1,22 @@
+
+  useEffect(() => {
+    setMounted(true);
+  }, []);
+
   return (
     <NavigationMenu suppressHydrationWarning>
       <NavigationMenuList suppressHydrationWarning>
         <NavigationMenuItem suppressHydrationWarning>
           <div className='w-fit' suppressHydrationWarning>
-            <WalletMultiButton className='!bg-gradient-to-r !from-blue-600 !to-purple-600 !border-0 !rounded-lg !px-6 !py-2 !text-white !font-medium !transition-all !duration-300 hover:!from-blue-700 hover:!to-purple-700 hover:!shadow-lg hover:!scale-105' />
+            {mounted && <WalletMultiButton className='bg-linear-to-r! from-blue-600! to-purple-600! border-0! rounded-lg! px-6! py-2! text-white! font-medium! transition-all! duration-300! hover:from-blue-700! hover:to-purple-700! hover:shadow-lg! hover:scale-105!' />}
```

ESLint

[error] Replace `<WalletMultiButton·className='bg-linear-to-r!·from-blue-600!·to-purple-600!·border-0!·rounded-lg!·px-6!·py-2!·text-white!·font-medium!·transition-all!·duration-300!·hover:from-blue-700!·hover:to-purple-700!·hover:shadow-lg!·hover:scale-105!'·/>` with `(⏎··············<WalletMultiButton·className='bg-linear-to-r!·from-blue-600!·to-purple-600!·border-0!·rounded-lg!·px-6!·py-2!·text-white!·font-medium!·transition-all!·duration-300!·hover:from-blue-700!·hover:to-purple-700!·hover:shadow-lg!·hover:scale-105!'·/>⏎············)`

(prettier/prettier)